scio-tensorflow/src/main/scala/com/spotify/scio/tensorflow/package.scala

```diff
@@ -0,0 +1,47 @@
+/*
+ * Copyright 2017 Spotify AB.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+package com.spotify.scio
+
+import com.spotify.scio.io.{FileStorage, TFFileStorageFunctions}
+import com.spotify.scio.values.{SCollection,
+                                TFRecordSCollectionFunctions,
+                                TensorFlowSCollectionFunctions}
+
+import scala.reflect.ClassTag
+
+package object tensorflow {
+
+  import scala.language.implicitConversions
+
+  /** Implicit conversion from [[SCollection]] to [[TensorFlowSCollectionFunctions]]. */
+  implicit def makeTensorFlowSCollectionFunctions[T: ClassTag](s: SCollection[T])
+  : TensorFlowSCollectionFunctions[T] = new TensorFlowSCollectionFunctions(s)
+
+  /** Implicit conversion from [[SCollection]] to [[TFRecordSCollectionFunctions]]. */
+  implicit def makeTFRecordSCollectionFunctions[T](s: SCollection[T])
+                                                  (implicit ev: T <:< Array[Byte])
+  : TFRecordSCollectionFunctions[T] = new TFRecordSCollectionFunctions(s)
+
+  /** Implicit conversion from [[ScioContext]] to [[TFRecordSCollectionFunctions]]. */
+  implicit def makeTFScioContextFunctions(s: ScioContext): TFScioConextFunctions =
```

Comment:
Just out of curiosity, what's the benefit of doing this instead of having an implicit class within this package object? Is it just so there's less code here?